pkg/test/kube/port_forwarder.go

```diff
@@ -83,20 +83,17 @@ func (f *defaultPortForwarder) Close() error {
 	return nil
 }
 
-func newPortForwarder(clientFactory util.Factory, pod v1.Pod, localPort, remotePort uint16) (PortForwarder, error) {
-	restClient, err := clientFactory.RESTClient()
+// NewPortForwarder creates a new PortForwarder to the given pod.
+func NewPortForwarder(restConfig *rest.Config, pod v1.Pod, localPort, remotePort uint16) (PortForwarder, error) {
```

Comment:
Yeah I went back and forth on this too. I left it in because I made Accessor an interface and since PortForwarder is also an interface, it seemed reasonable to allow the Accessor implementation to provide the PortForwarder implementation.